webapp/channels/src/plugins/export.js

```diff
@@ -47,6 +48,7 @@ window.StyledComponents = require('styled-components');
 window.PostUtils = {formatText, messageHtmlToComponent};
 window.openInteractiveDialog = openInteractiveDialog;
 window.useNotifyAdmin = useNotifyAdmin;
+window.sendDesktopNotificationToMe = notifyMe;
```

Comment:
Would you actually mind moving this into `window.WebappUtils` or `window.ProductApi`? I know we're not at all consistent about where we put these, but I prefer having them grouped together into some sort of object to avoid cluttering the global namespace with our own stuff (like how `window.Components` causes Firefox to print a warning on page load)

Reply:
Absolutely, done. 👍